src/crud.py

```
@@ -1,6 +1,6 @@
 from sqlalchemy.ext.asyncio import AsyncSession
 from sqlalchemy.future import select
-from sqlalchemy import func, and_, or_
+from sqlalchemy import func, and_, or_, delete
```

Ruff (0.11.9)

`sqlalchemy.or_` imported but unused

Remove unused import: `sqlalchemy.or_`

(F401)